src/opnsense/service/templates/OPNsense/Dnsmasq/dnsmasq.conf

```diff
@@ -115,7 +115,7 @@ conf-dir=/usr/local/etc/dnsmasq.conf.d,\*.conf
 {% if dhcp_range.start_addr and ':' not in dhcp_range.start_addr %}
 {# IPv4 range #}
 dhcp-range={%if dhcp_range.interface -%}
-tag:{{helpers.physical_interface(dhcp_range.interface)}},
+{% if dhcp_range.interface and dhcp_range.set_tag -%}tag:{%- endif -%}{{helpers.physical_interface(dhcp_range.interface)}},
```

Comment:
why is this? I was under the impression that we can always prepend `tag:` for interfaces as all requests are tagged by interface name, but maybe I'm missing something.

Reply:
let's prefix the `tag:` as this is more consistent. assuming the interface always receives a tag (which is also what I could find), setting it again shouldn't make much sense anyway.